packages/mds-policy-linter/utils.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 City of Los Angeles
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { PolicyDomainCreateModel } from '@mds-core/mds-policy-service'
+import { Nullable } from '@mds-core/mds-types'
+
+function describePolicy(policy: PolicyDomainCreateModel): string {
+  return policy.name + ' (' + policy.policy_id + ')'
+}
+
+const JAN_2020 = new Date(2020, 1, 1).getDate()
+/**
+ * is a date invalid?
+ * @param t a number or null or undefined
+ * @returns a string describing why a date is not legit, null for 'legit'
+ */
+function invalidDate(t: Nullable<number> | undefined): string | null {
+  if (typeof t !== 'number') {
+    return 'not a number'
+  }
+  if (t < JAN_2020) {
+    return 'too early'
+  }
+  return null
+}
+
+/**
+ * is a date valid?
+ * @param t a number or null or undefined
+ * @returns is the date valid, true or false
+ */
+function validDate(t: Nullable<number> | undefined): boolean {
+  return invalidDate(t) === null
+}
+
+function plural(n: number, s: string): string {
+  return '' + n + ' ' + (n === 1 ? s.replace(/ies$/, 'y').replace(/s$/, '') : s)
+}
```

Comment:
There's a very similar function already in `@mds-core/mds-utils` ... `pluralize(n, 'policy', 'policies')` is how it would be used.